Resources/config/schema/seo-1.0.xsd

```diff
@@ -18,10 +18,9 @@
         <xsd:attribute name="title" type="xsd:string" />
         <xsd:attribute name="description" type="xsd:string" />
         <xsd:attribute name="original-route-pattern" type="original_route_pattern" />
-        <xsd:attribute name="content-key" type="xsd:string" />
```

Comment:
another solution is to have 2 versions: 1.0 being the old one, 1.1 the new syntax. When going to 2.0, 1.0 is then going to be removed.